### ISSUE TITLE
Using semver ranges when specifying package version

### ISSUE THREAD

**Issue by andrewhughes101** [nodejs/citgm#762]
Came across the following error when trying to run citgm using semver ranges against a package when specifying the version.
```
citgm accepts@~1.3.0           
info:    starting            | accepts@~1.3.0      
warn:    update-available    | v5.0.8 (current: v5.0.3)
warn:                        | npm install -g citgm    
info:    lookup              | accepts             
info:    lookup-notfound     | accepts             
info:    accepts npm:        | Downloading project: accepts@~1.3.0
info:    accepts npm:        | Project downloaded accepts-1.3.7.tgz
info:    accepts npm:        | npm install started 
info:    accepts npm:        | npm install successfully completed
info:    accepts npm:        | test suite started  
error:   failure             | The canary is dead: 
error:   failing module(s)   |                     
error:   module name:        | accepts             
error:   version:            |                     
error:   error:              | The canary is dead: 
error:   error:              | undefinedadded 373 packages from 650 contributors and audited 956 packages in 6.303s                                      
error:                       | found 0 vulnerabilities                                                                                                   
error:                       |                                                                                                                           
error:                       |                                                                                                                           
error:                       | > accepts@1.3.7 test /private/var/folders/sm/wxmj4vf12gn4tysygy7gkjrm0000gn/T/14406050-2075-4064-a72a-8f32f7fb9a33/accepts
error:                       | > mocha --reporter spec --check-leaks --bail test/                                                                        
error:                       |                                                                                                                           
error:                       |                                                                                                                           
error:                       | Error: No test files found: "test/"                                                                                       
error:                       | npm ERR! Test failed.  See above for more details.                                                                        
error:   done                | The smoke test has failed.
info:    duration            | test duration: 9424ms
```

It seems to find v1.3.7 but still throws an error. Anyone know if this method of using citgm works?

node: 12.13.0
citgm: 5.0.6

**Comment by targos** [comment]
Proposed fix: https://github.com/nodejs/citgm/pull/763

**Comment by targos** [comment]
Released in v5.0.9
